pkg/network/event_windows.go

```diff
@@ -68,59 +66,43 @@ func getEphemeralRange(f ConnectionFamily, t ConnectionType) (low, hi uint16, er
 	default:
 		protoarg = "udp"
 	}
-	cmd := exec.Command("netsh", "int", familyarg, "show", "dynamicport", protoarg)
-	cmdOutput := &bytes.Buffer{}
+	output, err := exec.Command("netsh", "int", familyarg, "show", "dynamicport", protoarg).Output()
+	if err != nil {
+		return 0, 0, err
+	}
+	return parseNetshOutput(string(output))
+}
+
+func parseNetshOutput(output string) (low, hi uint16, err error) {
 	// output should be of the format
 	/*
 		Protocol tcp Dynamic Port Range
 		---------------------------------
 		Start Port      : 49000
 		Number of Ports : 16000
 	*/
-	cmd.Stdout = cmdOutput
-	err = cmd.Run()
-	if err != nil {
-		return
-	}
-	output := cmdOutput.Bytes()
-	var r = regexp.MustCompile(`.*: (\d+)`)
-
-	matches := r.FindAllStringSubmatch(string(output), -1)
+	matches := netshRegexp.FindAllStringSubmatch(output, -1)
```

Comment:
I'd probably rather have the regex defined here; easier to read what's going on if you can see the regex when looking at the submatch (and this will only be called once, anyway).  But I can live with it either way.